### ISSUE TITLE
Navit on Playstore

### ISSUE THREAD

**Issue by jkoan** [navit-gps/navit#1171]
Currently Navit for Android was removed from Google Playstore for the following reasons:

APK:2021100476 | Invalid privacy policyBased on our review, your privacy policy doesn’t comply with our policy requirements. Please review the Personal and Sensitive Information policy and ensure that you are compliant. Specifically, please add or update your privacy policy,  and make sure it is available on an active URL (no PDFs), is  non-editable, applies to your app, and specifically covers user privacy,  including your app’s usage of location data. If the URL provided points  to multiple privacy policies, please make any necessary external  changes (within the app, your website, and the Play Store listing - as  necessary) to point to one singular governing privacy policy. You must  link to a privacy policy on your app's store listing page and within  your app.
-- | --
APK:2021100476 | Prominent disclosure not foundYour app must display a prominent disclosure through a pop-up alert before your  app’s location runtime permission. Based on our review, a prominent  disclosure did not appear before the runtime permission.Please add a prominent disclosure before the runtime permission.Remember, your prominent disclosure must:  Appear before your app’s location runtime permission. Include at least the following sentence, adapted to include all the  relevant features requesting access to location in the background in the  app that are readily visible to the user: “This app collects  location data to enable ["feature"], ["feature"], & ["feature"] even  when the app is closed or not in use.” If you extend permitted usage to ads, please also include: “This data is also used to provide ads/support advertising/support ads.” Include any other details necessary to make it clear to the user how  and why you are using location in the background. While additional  content is permitted, it should not cause the required content to not be  immediately visible.

I think the way to go forward is to do the second point first as we can fix this within the app. For that I plan to rework the first time setup like we discussed here: https://app.diagrams.net/#Hnavit-gps%2Fui-mockups%2Fmaster%2Ffirst%20step.xml

The first point is a bit more interesting as we don't collect user data so we don't really need a privacy policy.
I added one at our docs in ed26d58ddcf7d48352cdd9f673f3e9096df0c905 but as this one is editable its probably not accepted. But strictly speaking every one we could publish will be editable, but probably one at the navit-gps/website should be fine.

**Comment by hoehnp** [comment]
@jkoan, the digramms look good. I just wonder if we should have some obvious explanation on the first screen about why we need these permissions?

**Comment by jkoan** [comment]
Yes we should definitely add this as its required by google and cant hurt anyway. The Mockup is already 3 Years old so there we didn't thought about this exact thing. So i would probably adjust the Mockup to have one page per permission with explanation for each
